setup.md

```diff
@@ -3,4 +3,15 @@ layout: page
 title: Setup
 root: .
 ---
-FIXME
+
+In order to be prepared for the lesson, you need to have accounts to the following (free) services:
+
+1. GitHub
+
+If you don't already have a [GitHub](https://github.com/) account, please follow the guide [here](https://services.github.com/on-demand/intro-to-github/create-github-account) in order to create one.
+
+2. BioTools
+
+[bio.tools](https://bio.tools/) is a portal to bioinformatics resources worldwide, aimed to help bioinformaticians and scientists, find, understand, compare and select resources as well as use and connect them in workflows.
```

Comment:
they don't need bio.tools account only dev.bio.tools

Reply:
We are actually explicitly requesting for participants to create an account on the dev instance in the next sentence. However, I thought it made sense to include an intro to the actual platform. If that is redundant, we can delete it.